src/pages/help-centre.js

```diff
@@ -1,13 +1,688 @@
+            <Li>
+                {localize(
+                    'Click on the "Reset your password link" near the bottom of the page.',
+                )}
+            </Li>
+            <Li>
+                {localize(
+                    'Enter your email address. You’ll receive a verification link in your inbox.',
+                )}
+            </Li>
+            <Li>
+                {localize(
+                    'Click on the verification link to set your new password.',
+                )}
+            </Li>
+        </Ol>
+        <Text>
+            {localize(
+                'If you have a real money account, you must also check the box labelled "I have a Real Money Account" and verify your date of birth.',
+            )}
+        </Text>
+    </>
+)
+
+const AuthenticatingYourAccount = () => (
+    <Text>
+        {localize('Authenticate your account on the verification page.')}
+    </Text>
+)
+
+const MakingADeposit = () => (
+    <>
+        <Text>
+            {localize(
+                'Click on the Cashier tab and choose your preferred deposit method.',
+            )}
+        </Text>
+        <Text>
+            {localize(
+                'Our list of supported payment methods includes bank wire, credit/debit cards, e-wallets, and cryptocurrencies.',
+            )}
+        </Text>
+        <Text>
+            {localize(
+                'You may also manage your funds through a payment agent if the service is available in your country.',
+            )}
+        </Text>
+        <Text>
+            {localize(
+                'You can deposit or withdraw a minimum of USD/EUR/GBP/AUD 5 using e-wallets. Other payment methods will have different minimum amounts. These amounts are stated on the payment methods page.',
+            )}
+        </Text>
+    </>
+)
+
+const ExpiredVerificationLink = () => (
+    <>
+        <Text>
+            {localize(
+                'We will occasionally prompt you to verify your account by sending you a verification link. This is to make sure it’s you and that your account hasn’t been compromised.',
+            )}
+        </Text>
+        <Text>
+            {localize(
+                'If your verification link has expired, try withdrawing or logging in again.',
+            )}
+        </Text>
+    </>
+)
+
+const WithdrawalProcessingTime = () => (
+    <Text>
+        {localize(
+            'We process all your deposits and withdrawals within 24 hours. However, your bank or money transfer service may require additional time to process your request.',
+        )}
+    </Text>
+)
+
+const articles = [
+    {
+        category: localize('Account'),
+        articles: [
+            {
+                title: localize('Who can open an account?'),
+                category: localize('Account'),
+                sub_category: localize('Opening an account'),
+                label: 'who-can-open-an-account',
+                content: <WhoCanOpenAnAccount />,
```

Comment:
All these links have underline text-decoration when you get hover on them. please check it is right or not.

Reply:
Yes should have